units/event_metajob.go

```diff
@@ -72,39 +70,26 @@ func makeEventMetaJob() *eventMetaJob {
 		},
 	}
 	j.SetDependency(dependency.NewAlways())
+	j.SetPriority(50)
```

Comment:
let's not escalate our priority system yet. most things are between -1 and 2, I'd put this as a 1...

Reply:
Oops, i only set that while testing. Switching to 1